websites/M/MEE6/presence.ts

```diff
@@ -2,59 +2,208 @@ const presence = new Presence({
+						presenceData.details = `Viewing plugin ${pathnameSplit[3]}`;
+						break;
+					}
+					case "monetize": {
+						presenceData.details = "Viewing monetisation options";
+						break;
+					}
+					case "custom-bot": {
+						presenceData.details = "Viewing custom bot options";
+						break;
+					}
+					case "pro":
+					case "premium": {
+						presenceData.details = "Viewing premium & pro options";
+						break;
+					}
+					case "careers": {
+						presenceData.details = "Viewing open careere positions";
+						break;
+					}
+					case "terms": {
+						presenceData.details = "Reading the terms";
+						presenceData.smallImageKey = Assets.Reading;
+						break;
+					}
+					case "privacy": {
+						presenceData.details = "Reading the privacy statement";
+						presenceData.smallImageKey = Assets.Reading;
+						break;
+					}
+					case "memberships": {
+						presenceData.details = "Managing their subscriptions";
+						break;
+					}
+					case "dashboard": {
+						switch (pathnameSplit[4]) {
+							case "": {
+								if (pathnameSplit[3] !== "") {
+									presenceData.details = document.querySelector(
+										'[class*="whitespace-nowrap !text-whit"]'
+									)?.textContent;
+									presenceData.largeImageKey = document
+										.querySelector('[class="sc-lmHNfd PQUIq"]')
+										.querySelector("img")
+										?.getAttribute("src");
+								} else presenceData.details = "Viewing the dashboard";
+								break;
+							}
+							default: {
+								if (
+									document.querySelector(
+										'[class="uppercase text-dark-400 text-xs font-semibold"]'
+									) &&
+									pathname.includes("commands")
+								) {
+									presenceData.details = "Editing custom command";
+									presenceData.state = await capitalizeFirstLetter(
+										document.querySelector('[id="item_header__name"]')
+											.textContent
+									);
+								} else if (pathname.includes("command/config/")) {
+									const command = document.querySelector(
+										'[class="sc-jdAMXn dmHLow sc-cmYsgE fhihFI"]'
+									);
+									if (!command) {
+										presenceData.details = "Loading";
+										presenceData.smallImageKey = Assets.Loading;
+									} else {
+										presenceData.details = "Editing command";
+										presenceData.state = command.textContent.slice(4);
+									}
+								} else {
+									presenceData.details = "Viewing module";
+									presenceData.state = await capitalizeFirstLetter(
+										document.querySelector(
+											'[class="font-bold text-dark-100 text-h5"]'
+										)?.textContent ?? pathnameSplit[4].trim()
+									);
+								}
+							}
+						}
+						break;
+					}
+					case "tutorials": {
+						switch (pathnameSplit[3]) {
+							// eslint-disable-next-line no-undefined
+							case undefined: {
+								presenceData.details = "Viewing all tutorials";
+								break;
+							}
+							default: {
+								presenceData.details = "Reading tutorials about:";
```

Comment:
Feels like this could be replaced by an if/else rather than a switch with disabling eslint

Reply:
Yea, if case would be better here, if (!pathnameSplit[3]) {} else {}